src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -12,3 +22,884 @@ pub mod crio;
+/// A sandbox annotation to enable rootless hypervisor (only supported in QEMU currently).
+pub const KATA_ANNO_CFG_HYPERVISOR_ENABLE_ROOTLESS_HYPERVISOR: &str =
+    "io.katacontainers.config.hypervisor.rootless";
+
+// Hypervisor Shared File System related annotations
+/// A sandbox annotation to specify the shared file system type, either virtio-9p or virtio-fs.
+pub const KATA_ANNO_CFG_HYPERVISOR_SHARED_FS: &str =
+    "io.katacontainers.config.hypervisor.shared_fs";
+/// A sandbox annotations to specify virtio-fs vhost-user daemon path.
+pub const KATA_ANNO_CFG_HYPERVISOR_VIRTIO_FS_DAEMON: &str =
+    "io.katacontainers.config.hypervisor.virtio_fs_daemon";
+/// A sandbox annotation to specify the cache mode for fs version cache or "none".
+pub const KATA_ANNO_CFG_HYPERVISOR_VIRTIO_FS_CACHE: &str =
+    "io.katacontainers.config.hypervisor.virtio_fs_cache";
+/// A sandbox annotation to specify the DAX cache size in MiB.
+pub const KATA_ANNO_CFG_HYPERVISOR_VIRTIO_FS_CACHE_SIZE: &str =
+    "io.katacontainers.config.hypervisor.virtio_fs_cache_size";
+/// A sandbox annotation to pass options to virtiofsd daemon.
+pub const KATA_ANNO_CFG_HYPERVISOR_VIRTIO_FS_EXTRA_ARGS: &str =
+    "io.katacontainers.config.hypervisor.virtio_fs_extra_args";
+/// A sandbox annotation to specify as the msize for 9p shares.
+pub const KATA_ANNO_CFG_HYPERVISOR_MSIZE_9P: &str = "io.katacontainers.config.hypervisor.msize_9p";
+
+// Runtime related annotations
+/// Prefix for Runtime configurations.
+pub const KATA_ANNO_CFG_RUNTIME_PREFIX: &str = "io.katacontainers.config.runtime.";
+/// A sandbox annotation that determines if seccomp should be applied inside guest.
+pub const KATA_ANNO_CFG_DISABLE_GUEST_SECCOMP: &str =
+    "io.katacontainers.config.runtime.disable_guest_seccomp";
+/// A sandbox annotation that determines if pprof enabled.
+pub const KATA_ANNO_CFG_ENABLE_PPROF: &str = "io.katacontainers.config.runtime.enable_pprof";
+/// A sandbox annotation that determines if experimental features enabled.
+pub const KATA_ANNO_CFG_EXPERIMENTAL: &str = "io.katacontainers.config.runtime.experimental";
+/// A sandbox annotaion that determines how the VM should be connected to the the container network
+/// interface.
+pub const KATA_ANNO_CFG_INTER_NETWORK_MODEL: &str =
+    "io.katacontainers.config.runtime.internetworking_model";
+/// SandboxCgroupOnly is a sandbox annotation that determines if kata processes are managed only in sandbox cgroup.
+pub const KATA_ANNO_CFG_SANDBOX_CGROUP_ONLY: &str =
+    "io.katacontainers.config.runtime.sandbox_cgroup_only";
+/// A sandbox annotation that determines if create a netns for hypervisor process.
+pub const KATA_ANNO_CFG_DISABLE_NEW_NETNS: &str =
+    "io.katacontainers.config.runtime.disable_new_netns";
+/// A sandbox annotation to specify how attached VFIO devices should be treated.
+pub const KATA_ANNO_CFG_VFIO_MODE: &str = "io.katacontainers.config.runtime.vfio_mode";
```

Comment:
Turning my head 90 degrees anti-clockwise to view this file, I'm strangely reminded of The Matrix screensaver :sunglasses: :red_circle: :large_blue_circle: :smile: